How to read review by phases (#1109)
italia/eidas-it-wallet-docs: docs/en/how-to-read-spec.rst

Reviewer: ```suggestion
This specification is designed to fulfil the requirements from multiple stakeholders within the IT-Wallet System. Each role has different responsibilities and scopes, and therefore different information needs. This section provides tailored reading paths to help you navigate efficiently to the content most relevant to the implementation goals.
```

Reviewer 2: ```suggestion
This specification is designed to fulfil the requirements from multiple stakeholders within the IT-Wallet System. Each role has different responsibilities and scopes, and therefore different information needs. This section provides tailored reading paths to help you navigate efficiently to the content most relevant to the implementation goals.
```

**Section** :ref:`appendix:Appendix`: 
  Provides supplementary technical details, implementation patterns, and testing frameworks including mobile application instance management, national platform integration specifications, and comprehensive test matrices for ecosystem validation.

Role based reading
^^^^^^^^^^^^^^^^^^

Quick paths by objective
^^^^^^^^^^^^^^^^^^^^^^^^

Use these summaries when you already know your goal; the role sections below expand each path.

> 

Reviewer: ```suggestion

```
Ridondante con sezione successiva, che ha già il titolo "Reading paths by roles"

Reading paths by roles
^^^^^^^^^^^^^^^^^^^^^^^^

For easier access to the topics covered in these specifications, role-based reading paths are presented below. Each path is organized according to the different stages of onboarding to and participation the IT-Wallet System, with direct references to the sections most relevant to each role. 

The proposed reading paths are intended as guidance and do not replace the need to consult other sections when a broader understanding of the system is required. 

For entities interested in addressing multiple roles, it is recommended to deepen all the reading paths related to relevant roles. 

> **Authentic Sources**

Reviewer: ```suggestion
**Authentic Source**
```

singolare

- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities. 

- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.

**Phase 2: Design**

To understand the requirements and design the features, functionalities and specific characteristics underlying a (Q)EAA to be issued.

- **Section** :ref:`functionalities:User Experience Design`: Key requirements on how to enable Users to obtain (Q)EAAs, (Q)EAA structures, status and management over time. 

- **Section** :ref:`digital-credential-management:Digital Credential Management`: Section digital-credential-management:Digital Credential Management: Technical and functional requirements related to (Q)EAA lifecycle. 
- **Section** :ref:`e-service-pdnd-template:PDND e-Service Template`: Standardized blueprint containing all necessary technical and descriptive metadata for the e-service definition.
**Phase 3: Implementation**

> To implement the technological interfaces required to communicate with Credential Issuers and tomanage the entire (Q)EAA data lifecycle.

Reviewer: ```suggestion
To implement the technological interfaces required to communicate with Credential Issuers and to manage the entire (Q)EAA data lifecycle.
```

typo

- **Section** :ref:`algorithms:Cryptographic Algorithms`: To meet security requirements by designing digital signature systems that make attestations unfalsifiable.

- **Section** :ref:`e-service-pdnd:e-Service PDND`: Mandatory integration specifications with the PDND (National Digital Data Platform) and the associated interoperability requirements for accessing authoritative data required for (Q)EAA issuance.
- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Credential Issuers in accordance with ISO/IEC 27001.
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem.
**Phase 4: Registration**

To become registered as Credential Issuer within the system, by completing the administrative and technical procedure so that (Q)EAA issued to the Wallet are officially trusted.

> - **Section** :ref:`onboarding-high-level:Onboarding System`: Overview of the onboarding system architecture and the Credential Issuer registration process.

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System`: Overview of the onboarding system architecture and the Credential Issuer registration process.

- **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Credential Issuer registration. 
```

Aggiunto secondo riferimento (entity onboarding) che era saltato

  Provides supplementary technical details, implementation patterns, and testing frameworks including mobile application instance management, national platform integration specifications, and comprehensive test matrices for ecosystem validation.

Reading paths by roles
^^^^^^^^^^^^^^^^^^^^^^^^

For easier access to the topics covered in these specifications, role-based reading paths are presented below. Each path is organized according to the different stages of onboarding to and participation the IT-Wallet System, with direct references to the sections most relevant to each role. 

The proposed reading paths are intended as guidance and do not replace the need to consult other sections when a broader understanding of the system is required. 

For entities interested in addressing multiple roles, it is recommended to deepen all the reading paths related to relevant roles. 

> **Authentic Source**

Reviewer: ```suggestion
### Authentic Source
```

> **Authentic Source**

Reviewer: ```suggestion
### Authentic Source
```

- **Section** :ref:`e-service-pdnd:e-Service PDND`: Mandatory integration specifications with the PDND (National Digital Data Platform) and the associated interoperability requirements for publishing an e-service. 
- **Section** :ref:`authentic-source-endpoint:Authentic Source Endpoints`: Key requirements for the implementation of APIs enabling the Credential Issuer to securely and consistently retrieve authoritative data and to manage the data lifecycle throught Signal Hub endpoint. 

- **Section** :ref:`registry:Registry Infrastructure`: Focus on Registry components of interest to the Authentic Source. 

- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and specific requirements for Authentic Sources in accordance with ISO/IEC 27001. 
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem.
**Phase 4: Registration**

To become registered as an Authentic Source within the system by completing the administrative and technical procedures required. 

> - **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Authentic Source registration process. 

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System`: Overview of the onboarding system architecture and the Authentic Source registration process. 
```

- **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Authentic Source registration.

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.

Reviewer: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations`: Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> **Wallet Provider**

Reviewer: ```suggestion
### **Wallet Provider**
```

The Wallet Provider focuses on designing and developing the Wallet Solution that enables the User to store, manage, and present their PID and (Q)EAAs. 

**Phase 1: Discovery**

To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust. 

- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System. 

- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes. 

- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities. 

- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.

**Phase 2: Design**

To understand the User Experience requirements and design the Wallet Solution following common patterns to ensure the usability and accessibility of the solutions.

> - **Section** :ref:`brand-identity:Brand Identity': Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Wallet Provider.  

Reviewer: ```suggestion
- **Section** :ref:`brand-identity:Brand Identity`: Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Wallet Provider.  
```

- **Section** :ref:`functionalities:User Experience Design`: Key requirements on Interaction Models and Interface layouts and graphic assets to ensure an effective and seamless User Experience and coherence among Wallet Solutions. 

- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to (Q)EAA lifecycle.

**Phase 3: Implementation**

To implement the Wallet Solution in line with specific technological standards to ensure the communication between the Wallet Solution and other actors.
- **Section** :ref:`wallet-solution:Wallet Solution`: Technical and functional requirements on components, functionalities and lifecycle to configure the Wallet Solution. 

> - **Section** :ref:`wallet-solution:Wallet Solution`: Technical and functional requirements on components, functionalities and lifecycle to configure the Wallet Solution. 

Reviewer: ```suggestion

```

Duplicato

- **Section** :ref:`digital-credential-flows:Digital Credential Flows`: Technical and functional requirements on (Q)EAA issuance and presentation flows. 

> - **Section** :ref:`digital-credential-flows:Digital Credential Flows`: Technical and functional requirements on (Q)EAA issuance and presentation flows. 

Reviewer: ```suggestion

```

Duplicato

Reviewer: riferimento alla sezione digital-credential-flows da eliminare.
Il commento che propone la cancellazione risulta risolto ma nella preview della PR è ancora visibile

- **Section** :ref:`wallet-provider-endpoint:Wallet Provider Endpoints`: Key requirements for the implementation of the Wallet Provider’s interfaces (APIs) required for interoperability among entities. 

> - **Section** :ref: `registry infrastructure`: Overview on Registry infrastructure and federation Registry. 

Reviewer: ```suggestion
- **Section** :ref:`registry infrastructure`: Overview on Registry infrastructure and federation Registry. 
```

> - **Section** :ref:`algorithms:Cryptographic Algorithms': Selection and implementation of cryptographic standards required to secure keys and transactions. 

Reviewer: ```suggestion
- **Section** :ref:`algorithms:Cryptographic Algorithms`: Selection and implementation of cryptographic standards required to secure keys and transactions. 
```

- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for Wallet Solutions.

- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Wallet Providers, in accordance with ISO/IEC 27001. 
- **Section** :ref:`mobile-application-instance: Mobile Application Instance`: Requirements for mobile application instance, related to initialization request and response. 

> - **Section** :ref:`mobile-application-instance: Mobile Application Instance`: Requirements for mobile application instance, related to initialization request and response. 

Reviewer: ```suggestion

```

Duplicato

- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem. 

> - **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem. 

Reviewer: ```suggestion

```

Duplicato

**Phase 4: Registration**

To become registered as a Wallet Provider within the system by completing the administrative and technical procedures so that the Wallet Solution is recognized by the system.
**Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Wallet Provider registration. 
- **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Wallet Provider registration process.  

> **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Wallet Provider registration. 

Reviewer: ```suggestion

```
Duplicato

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation

Reviewer: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations`: Operational procedures for managing X.509 Certificates within the IT-Wallet federation
```

> **Credential Issuer**

Reviewer: ```suggestion
### **Credential Issuer**
```

> **Credential Issuer**

Reviewer: ```suggestion
### **Credential Issuer**
```

**Phase 2: Design**

To understand the requirements and technically design (Q)EAA by structuring metadata to meet technical and regulatory requirements.

- **Section** :ref:`functionalities:User Experience Design`: Key requirements on (Q)EAA structure, issuance, status and management over time.

- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to(Q)EAA lifecycle.

**Phase 3: Implementation**

To develop endpoints based on specific protocols, and to implement (Q)EAA issuance, renewal, revocation and all the technical lifecycle management functions.
- **Section** :ref:`credential-issuer-solution:Credential Issuer Solution`: Technical and functional requirements on components, and interaction patterns to issue and manage (Q)EAAs lifecycle.
- **Section** :ref:`digital-credential-flows:Digital Credential Flows`: Technical and functional requirements on (Q)EAA issuance and presentation flows.

- **Section** :ref:`credential-issuer-endpoint:Credential Issuer Endpoints`: Key requirements for the implementation of Credential Issuer metadata and authorization endpoints.

> - **Section** :ref:`algorithms:Cryptographic Algorithms': To meet security requirements by designing digital signature systems that make attestations unfalsifiable.

Reviewer: ```suggestion
- **Section** :ref:`algorithms:Cryptographic Algorithms`: To meet security requirements by designing digital signature systems that make attestations unfalsifiable.
```

- **Section** :ref:`e-service-pdnd:e-Service PDND`: Mandatory integration specifications with the PDND (National Digital Data Platform) and the associated interoperability requirements for accessing authoritative data required for (Q)EAA issuance.
- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Credential Issuers in accordance with ISO/IEC 27001.
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem.
**Phase 4: Registration**

To become registered as Credential Issuer within the system, by completing the administrative and technical procedure so that (Q)EAA issued to the Wallet are officially trusted.

> - **Section** :ref:`entity-onboarding:Entity Onboarding': Focus on technical implementation procedures for Credential Issuer registration.

Reviewer: ```suggestion

```

> - **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Credential Issuer registration process.

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System`: Overview of the onboarding system architecture and the Credential Issuer registration process.
```

> - **Section** :ref:`entity-onboarding:Entity Onboarding': Focus on technical implementation procedures for Credential Issuer registration.

Reviewer: ```suggestion

```

Duplicato

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.

Reviewer: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations`: Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> - **Section** :ref:`entity-onboarding:Entity Onboarding': Focus on technical implementation procedures for Credential Issuer registration.

Reviewer: ```suggestion

```

Duplicato

> **Relying Party**

Reviewer: ```suggestion
### **Relying Party**
```

> **Relying Party**

Reviewer: ```suggestion
### **Relying Party**
```

The Relying Party focuses on securely requesting, receiving, and verifying the authenticity and validity of the PID and (Q)EAAs presented by the User to grant access to online and offline services.

**Phase 1: Discovery**

To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust.

- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System.

- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes.

- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities.

- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.

**Phase 2: Design**

To understand the User Experience requirements and design the verification functionalities necessary to provide the service to the end User.

> - **Section** :ref:`brand-identity:Brand Identity': Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Relying Party.

Reviewer: ```suggestion
- **Section** :ref:`brand-identity:Brand Identity`: Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Relying Party.
```

- **Section** :ref:`functionalities:User Experience Design`: Key requirements on Interaction Models, Interface layouts and graphic assets to ensure an effective and seamless User Experience, and coherence among presentation and verification systems.

- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to (Q)EAA lifecycle.

**Phase 3: Implementation**

To implement verification functionalities following specific protocols, to send verification requests to the Wallet and receive a response with the User’s authorization.
- **Section** :ref:`relying-party-solution:Relying Party Solution`: Technical and functional requirements on components and functionalities for PID and (Q)EAA verification.

> - **Section** ref:`digital-credential-flows:`Digital Credential Flows':  Technical and functional requirements on (Q)EAA presentation and verification flows.

Reviewer: ```suggestion
- **Section** ref:`digital-credential-flows:Digital Credential Flows`:  Technical and functional requirements on (Q)EAA presentation and verification flows.
```

- **Section** :ref:`relying-party-endpoints:Relying Party Endpoints`: Key requirements for the implementation of (Q)EAAs verification endpoints.

- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Relying Parties, in accordance with ISO/IEC 27001.
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem. 
**Phase 4: Registration**

To become registered as Relying Party within the system, by completing the administrative and technical procedure and be a reliable actor when requesting User’s data.

> - **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Relying Party registration process.

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System`: Overview of the onboarding system architecture and the Relying Party registration process.
```

- **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on implementation procedures for Relying Party registration. 

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.

Reviewer: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations`: Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> **Relying Party** — *Verify User Credentials:* :ref:`relying-party-solution:Relying Party Solution`, :ref:`credential-presentation:Digital Credential Presentation`, :ref:`relying-party-endpoints:Relying Party Endpoints`; *formats and validity:* :ref:`digital-credential-management:Digital Credential Management`; *security:* :ref:`security-privacy-considerations:Security and Privacy Considerations`.

Reviewer: ```suggestion

```

Approccio superato dal successivo (lettura per ruoli)

> Reading by project phase

Reviewer: ```suggestion
Reading paths by roles
```

^^^^^^^^^^^^^^^^^^^^^^^^

> For easier access to the topics in these specifications, a role-based reading path is presented below. This path considers the phase in which each role operates within the IT-Wallet system and includes direct references to the relevant sections of the document. 

Reviewer: ```suggestion
For easier access to the topics covered in these specifications, role-based reading paths are presented below. Each path is organized according to the different stages of onboarding to and participation the IT-Wallet System, with direct references to the sections most relevant to each role. 

The proposed reading paths are intended as guidance and do not replace the need to consult other sections when a broader understanding of the system is required. 

For entities interested in addressing multiple roles, it is recommended to deepen all the reading paths related to relevant roles. 
```

**Authentic Sources**

> The Authentic Source is the owner of the data. Their role is to ensure that original information is transmitted correctly and made securely available to issuing systems, and kept up to date. 

Reviewer: ```suggestion

```

> **Focus**: Data availability, accuracy of transmitted information, constant alignment between the original database and the issued credentials. 

Reviewer: ```suggestion
The Authentic Source focuses on securely exposing, managing, and guaranteeing the absolute accuracy and integrity of the authoritative raw data underlying (Q)EAA. 
```

**Phase 1: Discovery**

> **Objective**: To understand the reference context, technical requirements, and regulations necessary to become part of the Trust Chain upon which the entire system is founded. 

Reviewer: ```suggestion
To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust.
```

> - **Section** :ref:`introduction:Introduction`: Understanding the scope and regulatory language of the IT-Wallet ecosystem. 

Reviewer: ```suggestion
- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System. 
```

> - **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet system architecture in terms of governance and enabled operational processes. 

Reviewer: ```suggestion
- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes. 
```

> - **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the Trust Chain and registration in the Trust List to operate at both national and European levels. 

Reviewer: ```suggestion
- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities. 
```

> - **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Regulatory references, defined terms, and technical standards enabling secure and correct interoperability between all participants. (Use this section for any questions regarding terminology, reference standards, or acronyms). 

Reviewer: ```suggestion
- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.
```

> - **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Regulatory references, defined terms, and technical standards enabling secure and correct interoperability between all participants. (Use this section for any questions regarding terminology, reference standards, or acronyms). 

Reviewer: ```suggestion
- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.
```

**Phase 2: Design**

> **Objective**: To understand the implementation requirements for the attributes deemed necessary to translate data into a standardized digital format (EAA) recognized by the Wallet. 

Reviewer: ```suggestion
To understand the requirements and design the features, functionalities and specific characteristics underlying a (Q)EAA to be issued.
```

> - **Section** :ref:`functionalities:User Experience Design`: How users obtain Electronic Attestations (see paragraph :ref:`functionalities:Issuance of Electronic Attestations of Attributes'). 

Reviewer: ```suggestion
- **Section** :ref:`functionalities:User Experience Design`: Key requirements on how to enable Users to obtain (Q)EAAs, (Q)EAA structures, status and management over time. 
```

> - **Section** :ref:`entities:Entities`: Including the attributes necessary to provide the Electronic Attestation requested by the User (see paragraph :ref:`entities:Authentic Sources'). 

Reviewer: ```suggestion

```
Spostato in implementation

> 

Reviewer: commento superato con la revisione, da chiudere

> - **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical requirements for designing and managing the lifecycle of attestations. 

Reviewer: ```suggestion
- **Section** :ref:`digital-credential-management:Digital Credential Management`: Section digital-credential-management:Digital Credential Management: Technical and functional requirements related to (Q)EAA lifecycle. 
```

> 

Reviewer: ```suggestion
- **Section** :ref:`e-service-pdnd-template:PDND e-Service Template`: Standardized blueprint containing all necessary technical and descriptive metadata for the e-service definition.
```

**Phase 3: Implementation**

> **Objective**: To adopt the requirements needed to create the technological interfaces (APIs) required to communicate with Credential Issuers, managing the entire data lifecycle. 

Reviewer: ```suggestion
To implement the technological interfaces required to communicate with Credential Issuers and tomanage the entire (Q)EAA data lifecycle.
```

> 

Reviewer: ```suggestion
- **Section** :ref:`authentic-sources:Authentic Sources`: Authentic Source’s role and responsabilities.

- **Section** :ref:`e-service-pdnd:e-Service PDND`: Mandatory integration specifications with the PDND (National Digital Data Platform) and the associated interoperability requirements for publishing an e-service. 
```

> - **Section** :ref:`endpoints:Endpoints`: Implementation of APIs enabling the Credential Issuer to securely and consistently retrieve authoritative data (see paragraph :ref:`endpoints:Authentic Source Endpoints'). 

Reviewer: ```suggestion
- **Section** :ref:`authentic-source-endpoint:Authentic Source Endpoints`: Key requirements for the implementation of APIs enabling the Credential Issuer to securely and consistently retrieve authoritative data and to manage the data lifecycle throught Signal Hub endpoint. 
```

> - **Section** :ref:`digital-credential-management:Digital Credential Management`: Lifecycle and management of Electronic Attestations (with particular reference to :ref:`digital-credential-management:Digital Credential Lifecycle'). 

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
- **Section** :ref:`registry:Registry Infrastructure`: Focus on Registry components of interest to the Authentic Source. 
```

> - **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

Reviewer: ```suggestion

```

> - **Section** :ref:`log-retention-policy:General Log Retention Policies`: Log retention requirements in accordance with ISO/IEC 27001 

Reviewer: ```suggestion
- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and specific requirements for Authentic Sources in accordance with ISO/IEC 27001. 
```

> 

Reviewer: ```suggestion
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem.
```

**Phase 4: Registration**

> **Objective**: To become accredited as an Authentic Source within the system by following the administrative and technical procedures that guarantee the entity's reliability. 

Reviewer: ```suggestion
To become registered as an Authentic Source within the system by completing the administrative and technical procedures required. 
```

> - **Section** :ref:`onboarding-high-level:Onboarding System': Participation in the IT-Wallet ecosystem and data recognition across all actors (see paragraph :ref:`onboarding-high-level:Authentic Source Registration Process'). 

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Authentic Source registration process. 
```

> 

Reviewer: ```suggestion
- **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Authentic Source registration.
```

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation

Reviewer: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> 

Reviewer: commento superato con la revisione, da chiudere

**Wallet Provider**

> The Wallet Provider is the entity that designs the digital wallet, whether public or private. They serve as the direct point of contact for the citizen; their challenge is to combine a fluid and intuitive user experience with the highest security standards. 

Reviewer: ```suggestion

```

> **Focus**: Compliance with protocols and security standards, fluidity of the user experience, adherence to IT-Wallet system guidelines. 

Reviewer: ```suggestion
The Wallet Provider focuses on designing and developing the Wallet Solution that enables the User to store, manage, and present their PID and (Q)EAAs. 
```

**Phase 1: Discovery**

> **Objective**: To understand the reference context, technical requirements, and regulations necessary to become part of the Trust Chain.  

Reviewer: ```suggestion
To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust. 
```

> - **Section** :ref:`introduction:Introduction`: Understanding the scope and regulatory language of the IT-Wallet ecosystem. 

Reviewer: ```suggestion
- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System. 
```

> - **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet system architecture in terms of governance and enabled operational processes. 

Reviewer: ```suggestion
- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes. 
```

> - **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the Trust Chain and registration in the Trust List to operate at both national and European levels. 

Reviewer: ```suggestion
- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities. 
```

> - **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Regulatory references, defined terms, and technical standards enabling secure and correct interoperability between all participants. (Use this section for any questions regarding terminology, reference standards, or acronyms). 

Reviewer: ```suggestion
- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.
```

**Phase 2: Design**

> **Objective**: To define the Wallet's visual identity, design the user experience, model the technical entity, and select the cryptographic algorithms that will ensure data protection on the device. 

Reviewer: ```suggestion
To understand the User Experience requirements and design the Wallet Solution following common patterns to ensure the usability and accessibility of the solutions.
```

> - **Section** :ref:`brand-identity:Brand Identity': Ensuring the App complies with communication guidelines and the visual identity of the IT-Wallet system. 

Reviewer: ```suggestion
- **Section** :ref:`brand-identity:Brand Identity': Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Wallet Provider.  
```

> - **Section** :ref:`functionalities:User Experience Design`: Designing simple and secure interfaces, ensuring an immediate and intuitive user experience. 

Reviewer: ```suggestion
- **Section** :ref:`functionalities:User Experience Design`: Key requirements on Interaction Models and Interface layouts and graphic assets to ensure an effective and seamless User Experience and coherence among Wallet Solutions. 
```

> - **Section** :ref:`entities:Entities`: Implementation requirements and attributes needed to configure the entity (see paragraph :ref:`entities:Wallet Solution'). 

Reviewer: ```suggestion

```
Spostata in implementation

> - **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical requirements for designing and managing the lifecycle of attestations. 

Reviewer: ```suggestion
- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to (Q)EAA lifecycle.
```

> - **Section** :ref:`algorithms:Cryptographic Algorithms': Selection and implementation of cryptographic standards required to secure keys and transactions. 

Reviewer: ```suggestion

```
Spostato in implementation

**Phase 3: Implementation**

> **Objective**: To implement communication endpoints and adopt technological standards (e.g., SD-JWT) that ensure the Wallet can interface with other actors.

Reviewer: ```suggestion
To implement the Wallet Solution in line with specific technological standards to ensure the communication between the Wallet Solution and other actors.
```

> 

Reviewer: ```suggestion
- **Section** :ref:`wallet-solution:Wallet Solution`: Technical and functional requirements on components, functionalities and lifecycle to configure the Wallet Solution. 

- **Section** :ref:`digital-credential-flows:Digital Credential Flows`: Technical and functional requirements on (Q)EAA issuance and presentation flows. 
```

> - **Section** :ref:`endpoints:Endpoints`: Implementation of the Wallet Provider’s interfaces (APIs) required for system interoperability (see paragraph :ref:`endpoints:Wallet Provider Endpoints'). 

Reviewer: ```suggestion
- **Section** :ref:`wallet-provider-endpoint:Wallet Provider Endpoints`: Key requirements for the implementation of the Wallet Provider’s interfaces (APIs) required for interoperability among entities. 
```

> 

Reviewer: ```suggestion
- **Section** :ref: `registry infrastructure`: Overview on Registry infrastructure and federation Registry. 

- **Section** :ref:`algorithms:Cryptographic Algorithms': Selection and implementation of cryptographic standards required to secure keys and transactions. 
```

> - **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

Reviewer: ```suggestion
- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for Wallet Solutions.
```

> - **Section** :ref:`log-retention-policy:General Log Retention Policies`: Log retention requirements in accordance with ISO/IEC 27001 

Reviewer: ```suggestion
- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Wallet Providers, in accordance with ISO/IEC 27001. 
```

> 

Reviewer: ```suggestion
- **Section** :ref:`mobile-application-instance: Mobile Application Instance`: Requirements for mobile application instance, related to initialization request and response. 

- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem. 
```

**Phase 4: Registration**

> **Objective**: To become accredited as a Wallet Provider so that citizens' Wallet Instances are recognized as valid by the system.  

Reviewer: ```suggestion
To become registered as a Wallet Provider within the system by completing the administrative and technical procedures so that the Wallet Solution is recognized by the system.
```

> - **Section** :ref:`onboarding-high-level:Onboarding System': Participation methods and data recognizability (see paragraph :ref:`onboarding-high-level:Wallet Provider Operator Journey'). 

Reviewer: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Wallet Provider registration process.  
```

> 

Reviewer: ```suggestion
**Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on technical implementation procedures for Wallet Provider registration. 
```

- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation

> - **Section**: :ref:`est-plans:Test Plans': Information on how test environments support implementers, auditors, and compliance testing in validating Wallet Solution behavior. 

Reviewer: ```suggestion

```

> - **Section**: :ref:`est-plans:Test Plans': Information on how test environments support implementers, auditors, and compliance testing in validating Wallet Solution behavior. 

Reviewer: commento superato con la revisione, da chiudere

**Credential Issuer**

The Credential Issuer is the technical provider of credentials. Its main task is to transform data received from Authentic Sources into secure digital attestations, ready to be issued to the Wallet. It is the technical hub ensuring interoperability through international standards like OpenID4VCI. 

> **Focus**: Designing credentials from raw data, security of obtained data, credential lifecycle. 

Reviewer 3: ```suggestion
The Credential Issuer focuses on transforming authoritative raw data from Authentic Source into (Q)EAAs, and managing their entire lifecycle from issuance and to revocation or expiration.
```

**Phase 1: Discovery**

> **Objective**: To understand the reference context, technical requirements, and regulations to join the Trust Chain.

Reviewer 3: ```suggestion
To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust.
```

> - **Section** :ref:`introduction:Introduction`: Understanding the scope and regulatory language of the IT-Wallet ecosystem. 

Reviewer 3: ```suggestion
- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System. 
```

> - **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet system architecture in terms of governance and enabled operational processes. 

Reviewer 3: ```suggestion
- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes. 
```

> - **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the Trust Chain and registration in the Trust List to operate at both national and European levels. 

Reviewer 3: ```suggestion
- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities.
```

> - **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Regulatory references, defined terms, and technical standards enabling secure and correct interoperability between all participants. (Use this section for any questions regarding terminology, reference standards, or acronyms).

Reviewer 3: ```suggestion
- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.
```

**Phase 2: Design**

> ****Objective**: To technically design attestations by structuring metadata to meet technical and regulatory requirements, allowing the user to share only necessary information. 

Reviewer 3: ```suggestion
To understand the requirements and technically design (Q)EAA by structuring metadata to meet technical and regulatory requirements.
```

> - **Section** :ref:`functionalities:User Experience Design`: For high-level functional requirements supporting the User Experience during all interaction phases. 

Reviewer 3: ```suggestion
- **Section** :ref:`functionalities:User Experience Design`: Key requirements on (Q)EAA structure, issuance, status and management over time.
```

> - **Section** :ref:`entities:Entities`: Implementation requirements and attributes for configuring the Issuer profile (see paragraphs :ref:`entities:Credential Issuer Solution', and :ref:`entities:Authentic Sources')

Reviewer 3: ```suggestion

```
Moved into Implementation phase

> - **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical requirements for designing and managing the lifecycle of attestations. 

Reviewer 3: ```suggestion
- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to(Q)EAA lifecycle.
```

> - **Section** :ref:`algorithms:Cryptographic Algorithms': To meet security requirements by designing digital signature systems that make attestations unfalsifiable. 

Reviewer 3: ```suggestion

```
Moved into Implementation phase

**Phase 3: Implementation**

> **Objective**: To develop issuing endpoints based on the OpenID4VCI protocol, implementing release, renewal, and technical lifecycle management functions.

Reviewer 3: ```suggestion
To develop endpoints based on specific protocols, and to implement (Q)EAA issuance, renewal, revocation and all the technical lifecycle management functions.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`credential-issuer-solution:Credential Issuer Solution`: Technical and functional requirements on components, and interaction patterns to issue and manage (Q)EAAs lifecycle.
```

> - **Section** ref:`digital-credential-flows:`Digital Credential Flows': Detailed specifications on implementing issuance flows, presentation methods, and wallet information retrieval. 

Reviewer 3: ```suggestion
- **Section** :ref:`digital-credential-flows:Digital Credential Flows`: Technical and functional requirements on (Q)EAA issuance and presentation flows.
```

> - **Section** ref:`digital-credential-flows:`Digital Credential Flows': Detailed specifications on implementing issuance flows, presentation methods, and wallet information retrieval. 

Reviewer 3: commento obsoleto, da chiudere

> - **Section** :ref:`endpoints:Endpoints`: Implementation of issuance interfaces, including the Credential Endpoint and Token Endpoint (see paragraph :ref:`endpoints:`Credential Issuer Endpoints'). 

Reviewer 3: ```suggestion
- **Section** :ref:`credential-issuer-endpoint:Credential Issuer Endpoints`: Key requirements for the implementation of Credential Issuer metadata and authorization endpoints.
```

> - **Section** :ref:`endpoints:Endpoints`: Implementation of issuance interfaces, including the Credential Endpoint and Token Endpoint (see paragraph :ref:`endpoints:`Credential Issuer Endpoints'). 

Reviewer 3: commento obsoleto, da chiudere

> 

Reviewer 3: ```suggestion
- **Section** :ref:`algorithms:Cryptographic Algorithms': To meet security requirements by designing digital signature systems that make attestations unfalsifiable.

- **Section** :ref:`e-service-pdnd:e-Service PDND`: Mandatory integration specifications with the PDND (National Digital Data Platform) and the associated interoperability requirements for accessing authoritative data required for (Q)EAA issuance.
```

- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

> - **Section** :ref:`log-retention-policy:General Log Retention Policies`: Log retention requirements in accordance with ISO/IEC 27001

Reviewer 3: ```suggestion
- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Credential Issuers in accordance with ISO/IEC 27001.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem.
```

**Phase 4: Registration**

> **Objective**: To become accredited so that credentials issued to the Wallet are officially "trusted" and verifiable.  

Reviewer 3: ```suggestion
To become registered as Credential Issuer within the system, by completing the administrative and technical procedure so that (Q)EAA issued to the Wallet are officially trusted.
```

> - **Section** :ref:`onboarding-high-level:Onboarding System': Participation methods and data recognizability (see paragraph :ref:`onboarding-high-level:Credential Issuer Operator Journey'). 

Reviewer 3: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Credential Issuer registration process.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`entity-onboarding:Entity Onboarding': Focus on technical implementation procedures for Credential Issuer registration.
```

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation

Reviewer 3: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> - **Section**: :ref:`est-plans:Test Plans': Information on how test environments support implementers, auditors, and compliance testing in validating Wallet Solution behavior. 

Reviewer 3: ```suggestion

```
Moved into Implementation phase

**Relying Party**

The Relying Party is the end-user of the credentials. Its task is to transform digital verification into business value, while ensuring a fast, secure identification process that complies with data minimization principles. 

> **Focus**: Designing the credential verification experience, respecting user privacy. 

Reviewer 3: ```suggestion
The Relying Party focuses on securely requesting, receiving, and verifying the authenticity and validity of the PID and (Q)EAAs presented by the User to grant access to online and offline services.
```

**Phase 1: Discovery**

> **Objective**: To understand the context and regulations to join the Trust Chain.

Reviewer 3: ```suggestion
To understand the general functioning of the ecosystem, the technical architecture and the Infrastructure of Trust.
```

> - **Section** :ref:`introduction:Introduction`: Understanding the scope and regulatory language of the IT-Wallet ecosystem. 

Reviewer 3: ```suggestion
- **Section** :ref:`introduction:Introduction`: Scope and regulatory context of the IT-Wallet System.
```

> - **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet system architecture in terms of governance and enabled operational processes. 

Reviewer 3: ```suggestion
- **Section** :ref:`architecture-overview:Architecture Overview`: Overview of the IT-Wallet System architecture in terms of governance and enabled operational processes.
```

> - **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the Trust Chain and registration in the Trust List to operate at both national and European levels. 

Reviewer 3: ```suggestion
- **Section** :ref:`trust-infrastructure:The Infrastructure of Trust`: Key requirements of the federation-based trust model and trust evaluation mechanisms between entities.
```

> - **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Regulatory references, defined terms, and technical standards enabling secure and correct interoperability between all participants. (Use this section for any questions regarding terminology, reference standards, or acronyms).

Reviewer 3: ```suggestion
- **Section** :ref:`defined-terms-and-references:Defined Terms and References`: Comprehensive terminology, normative references, additional documentation, tools, resources and contribution guidelines.
```

**Phase 2: Design**

> ****Objective**: To define verification requirements by establishing the attributes necessary to provide the service, and to design the request experience by defining data-sharing methods.  

Reviewer 3: ```suggestion
To understand the User Experience requirements and design the verification functionalities necessary to provide the service to the end User.
```

> - **Section** :ref:`brand-identity:Brand Identity': Ensuring that any verification applications comply with the communication guidelines and visual identity of the IT-Wallet system. 

Reviewer 3: ```suggestion
- **Section** :ref:`brand-identity:Brand Identity': Overview of the IT-Wallet Brand Identity and indications on assets to be adopted by the Relying Party.
```

> - **Section** :ref:`functionalities:User Experience Design`: High-level functional requirements supporting the user experience across all interaction phases between the User and the service.  

Reviewer 3: ```suggestion
- **Section** :ref:`functionalities:User Experience Design`: Key requirements on Interaction Models, Interface layouts and graphic assets to ensure an effective and seamless User Experience, and coherence among presentation and verification systems.
```

> - **Section** :ref:`entities:Entities`: Implementation requirements and attributes necessary to configure the Verifier profile (see paragraph :ref:`entities:Relying Party Solution').

Reviewer 3: ```suggestion

```
Moved into Implementation phase

> - **Section** :ref:`digital-credential-management:Digital Credential Management`: Formats of Electronic Credentials and methods for verifying their validity. 

Reviewer 3: ```suggestion
- **Section** :ref:`digital-credential-management:Digital Credential Management`: Technical and functional requirements related to (Q)EAA lifecycle.
```

> - **Section** :ref:`algorithms:Cryptographic Algorithms': Design and implementation of digital signature validation systems ensuring the integrity of the data presented by the user. 

Reviewer 3: ```suggestion

```
Moved into Implementation phase

**Phase 3: Implementation**

> **Objective**: To implement the OpenID4VP protocol to send requests to the Wallet and receive attestations, ensuring correct format decoding. 

Reviewer 3: ```suggestion
To implement verification functionalities following specific protocols, to send verification requests to the Wallet and receive a response with the User’s authorization.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`relying-party-solution:Relying Party Solution`: Technical and functional requirements on components and functionalities for PID and (Q)EAA verification.
```

> - **Section** ref:`digital-credential-flows:`Digital Credential Flows':  Implementation of presentation flows for both remote and proximity scenarios (see paragraph ref:`digital-credential-flows:`Digital Credential Presentation'). 

Reviewer 3: ```suggestion
- **Section** ref:`digital-credential-flows:`Digital Credential Flows':  Technical and functional requirements on (Q)EAA presentation and verification flows.
```

> - **Section** :ref:`endpoints:Endpoints`: Implementation of receiving and verification interfaces (see paragraphs :ref:`endpoints:`Relying Party Endpoints' and :ref:`endpoints:Relying Party Provider Backend Endpoints'). 

Reviewer 3: ```suggestion
- **Section** :ref:`relying-party-endpoints:Relying Party Endpoints`: Key requirements for the implementation of (Q)EAAs verification endpoints.
```

- **Section** :ref:`security-privacy-considerations:Security and Privacy Considerations`: Security and compliance requirements for implemented solutions. 

> - **Section** :ref:`log-retention-policy:General Log Retention Policies`: Log retention requirements in accordance with ISO/IEC 27001

Reviewer 3: ```suggestion
- **Section** :ref:`log-retention-policy:General Log Retention Policies`: General Log retention requirements and requirements specific for Relying Parties, in accordance with ISO/IEC 27001.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`test-plans:Test Plans`: Guide to set up the test environment and validate backend interactions with the test matrices provided by the ecosystem. 
```

**Phase 4: Registration**

> **Objective**: To become accredited as a secure and reliable actor when requesting user data.   

Reviewer 3: ```suggestion
To become registered as Relying Party within the system, by completing the administrative and technical procedure and be a reliable actor when requesting User’s data.
```

> - **Section** :ref:`onboarding-high-level:Onboarding System': learning the methods for participating in the IT-Wallet ecosystem  (see paragraph :ref:`ronboarding-high-level:Relying Party Operator Journey'). 

Reviewer 3: ```suggestion
- **Section** :ref:`onboarding-high-level:Onboarding System': Overview of the onboarding system architecture and the Relying Party registration process.
```

> 

Reviewer 3: ```suggestion
- **Section** :ref:`entity-onboarding:Entity Onboarding`: Focus on implementation procedures for Relying Party registration. 
```

> - **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation

Reviewer 3: ```suggestion
- **Section** :ref:`x5c-evaluation:X.509 Certificate Management Operations': Operational procedures for managing X.509 Certificates within the IT-Wallet federation.
```

> - **Section**: :ref:`est-plans:Test Plans': Information on how test environments support implementers, auditors, and compliance testing in validating Wallet Solution behavior. 

Reviewer 3: ```suggestion

```
Moved into Implementation phase

> - **Section**: :ref:`est-plans:Test Plans': Information on how test environments support implementers, auditors, and compliance testing in validating Wallet Solution behavior. 

Reviewer 3: commento obsoleto, da chiudere

> NOTE

Reviewer 3: ```suggestion

```

> For implementers working on solutions that cover multiple roles (e.g., a combination of Credential Provider Solutions and Relying Parties), it is recommended to review the sections for all relevant roles before proceeding with development. It is important to pay particular attention to Entity Configuration requirements and to federation flows that apply across multiple roles.

Reviewer 3: ```suggestion

```
Moved into the introduction of this section.

> For implementers working on solutions that cover multiple roles (e.g., a combination of Credential Provider Solutions and Relying Parties), it is recommended to review the sections for all relevant roles before proceeding with development. It is important to pay particular attention to Entity Configuration requirements and to federation flows that apply across multiple roles.

Reviewer 3: commento obsoleto, da chiudere
